deployer/src/handlers/mod.rs

```diff
@@ -621,6 +623,19 @@ pub async fn start_deployment(
         };
         deployment_manager.run_push(built).await;
 
+        tokio::spawn(async move {
+            let event =
+                async_posthog::Event::new("shuttle_api_start_deployment".to_string(), account_name);
+            match deployment_manager.posthog_client().capture(event).await {
+                Ok(_) => {
+                    info!("Event sent to posthog");
+                }
+                Err(err) => {
+                    error!(error = %err, "failed to send event to posthog");
```

Comment:
I don't think we should log anything in the Ok path, we're covered with just the error event. So we can do `if let Err(err) = posthog_client()...` here.